runtime/queries/bash/injections.scm

```diff
@@ -1,3 +1,11 @@
 (heredoc_redirect
   (heredoc_body) @injection.content
   (heredoc_end) @injection.language)
+
+(redirected_statement
+  body: (command
+    argument: (string
+      (string_content) @injection.content))
+  redirect: (file_redirect
+    destination: (word) @injection.filename)
```

Comment:
There should be a limit to whether it's `>` or `<`, or something else.

Reply:
Just to be sure

```bash
echo "foo" > bar < bar.lua
```

```
program [0, 0] - [1, 0]
  redirected_statement [0, 0] - [0, 26]
    body: command [0, 0] - [0, 10]
      name: command_name [0, 0] - [0, 4]
        word [0, 0] - [0, 4]
      argument: string [0, 5] - [0, 10]
        string_content [0, 6] - [0, 9]
    redirect: file_redirect [0, 11] - [0, 16]
      destination: word [0, 13] - [0, 16]
    redirect: file_redirect [0, 17] - [0, 26]
      destination: word [0, 19] - [0, 26]
```

Reply:
As I don't bash, please make a concrete suggestion.

Reply:
I don't bash as well but I believe this can lead to duplicated injections (my example above) so it's better to limit it to one stream only (probably stdout?)

Reply:
Probably something like the below would be better. @lewis6991 
```suggestion
  redirect: (file_redirect
    ">"
    destination: (word) @injection.filename)
```

Reply:
I'm not really sure that this is a big problem. Maybe just add a todo?

Reply:
>     echo "foo" > bar < bar.lua

Given that doesn't work I don't think we need to worry about it.